db/seeds.d/101-locations.rb

```diff
@@ -4,9 +4,8 @@
 # !!! PLEASE KEEP THIS SCRIPT IDEMPOTENT !!!
 #
 
-if Location.exists? &&
-    Setting[:default_location_subscribed_hosts].blank? ||
-    Setting[:default_location_puppet_content].blank?
+if Location.exists? && && (ENV['SEED_LOCATION'].present? || Location.first.present?) &&
+    (Setting[:default_location_subscribed_hosts].blank? || Setting[:default_location_puppet_content].blank?)
```

Comment:
This is getting a bit tricky, currently to seed the location, one of these would now have to be `blank?` which is not something I think we want to gate the seeding of the default location based on. When I look at line 15 and line 18 I am not entirely sure why these checks are here.

This seems to come from (https://github.com/jlsherrill/katello/commit/c3a0892d1dfe3848d1c27ec1a038bdcc2d2cfd53#diff-eeab032f7fcfdf3e5414a140115e27f4) but even that is not obvious to me in which cases we want to allow blank default.